### PR TITLE
doc: fixed readme

### PR DESCRIPTION
**Pull request by karasusan**
- fixed roadmap
- added doc how to build plugin on Linux

### CODE REVIEW

File: Plugin~/README_EN.md [hunk 1]
```diff
@@ -1,32 +1,120 @@
-# Building the Plugin
+# Native Plugin
 
 This guide will cover building and deploying the native plugin `com.unity.webrtc` depends on.
 
-### Embedding libwebrtc
+## Developing environment
+
+Install dependencies libraries to make developing environment.
```
Comment (sindharta) [inline Plugin~/README_EN.md]: Install dependencies to create the development environment

File: Plugin~/README_EN.md [hunk 2]
```diff
@@ -1,32 +1,120 @@
...
+wget https://vulkan.lunarg.com/sdk/download/1.1.121.2/windows/VulkanSDK-1.1.121.2-Installer.exe -O C:/Windows/Temp/VulkanSDK.exe
+C:/Windows/Temp/VulkanSDK.exe /S
+
+# Setting up environment variables
+setx CUDA_PATH "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1" /m
+setx VULKAN_SDK "C:\VulkanSDK\1.1.121.2" /m
+```
 
-The plugin relies on **libwebrtc**, so building it requires a static libwebrtc link. `webrtc-win.zip` can be found on the Github Release page.
+### How to install dependencies libraries (Ubuntu18.04)
```
Comment (sindharta) [inline Plugin~/README_EN.md]: "How to install dependencies"

"Libraries" is not required

File: Plugin~/README_EN.md [hunk 3]
```diff
@@ -1,32 +1,120 @@
...
+
+# Install CUDA SDK
+sudo apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub
+wget http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.1.243-1_amd64.deb
+sudo dpkg -i cuda-repo-ubuntu1804_10.1.243-1_amd64.deb
+sudo apt update
+sudo apt install -y cuda
+```
+
+### How to install dependencies libraries (MacOS)
```
Comment (sindharta) [inline Plugin~/README_EN.md]: This one too

**Comment by karasusan** [comment]
Related issue.
https://github.com/Unity-Technologies/com.unity.webrtc/issues/41

**Pull request merged**